### ISSUE TITLE
No update due to too long. Most Integration plugins are no longer working

### ISSUE THREAD

**Issue by BUGProtectionAssociation** [Mixaill/awesome-gog-galaxy#126]
GoG can almost exclusively connect Epic and Xbox. Everything else has big problems

**Comment by Mixaill** [comment]
TLDR: GOG failed to communicate with the community, provide new features and moderate updates, so the community lost interest in the zombie launcher.

**Comment by Destroy666x** [comment]
I'm having issues only with PSN and Ubisoft, but both are regular and require manual patches or fork replacements, so I'm switching to Playnite right now. It also has integrations that GOG doesn't, e.g. Nintendo. Too bad you need yet another app because GOG failed to be what it wanted to be, but what can you do.

**Comment by fl4shback** [comment]
> TLDR: GOG failed to communicate with the community, provide new features and moderate updates, so the community lost interest in zombie launcher.

Yeah, that pretty much sums it up. I'm back on Playnite with Stardust theme (galaxy inspired), it's less user friendly but can't really say Galaxy is user friendly anymore when users have to dig through python files to fix integrations themselves.

I'm pretty sure Galaxy is almost abandoned at this point. Minimal updates to keep the GoG part alive. Doesn't seem like they have any interest in integrations anymore.

**Comment by BUGProtectionAssociation** [comment]
This object is dead. It has no time! Goodbye!
